docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,69 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
+
+Image Selection
+---------------
+
+- [ ] Cancel image selection dialog
+- [ ] Select an unbootable image (without a partition table)
+- [ ] Attempt to select an unsupported image
+- [ ] Change image selection
+
+Drive Selection
+---------------
+
+- [ ] Open the drive selection modal
+
+Image Support
+-------------
+
+- [ ] Flash an uncompressed image
+- [ ] Flash a Bzip2 image
+- [ ] Flash a XZ image
+- [ ] Flash a ZIP image
```

Comment:
Can we add gzipped & dmg images here as well?

Reply:
Yeah, will do.

Reply:
BTW, do you think its worth uploading some example files for this format to Google Drive, and linking them here?

Reply:
We have a bunch of them in `tests/image-stream/data` – might be worth mentioning those as a source

Reply:
> We have a bunch of them in `tests/image-stream/data`

Which also contains some zip images containing metadata (icon etc.) and bmaps, which aren't mentioned in this list yet.